admin/src/app/foms/fom-submission/submission-format-overview.component.html

```diff
@@ -1,12 +1,9 @@
 
-<mat-expansion-panel hideToggle>
+<mat-expansion-panel style="padding: 2px;">
```

Comment:
Hi Ian, I just wondering for these small styling change, do I need to add a classname and add to css file? or it's okay to do inline style?

Reply:
It's not consistent in FOM for this.
However, most of the time I strive to use class name, but when found very minor, then could go with just css style.
Could we try using class name for this expansion-panel? or want to improve it by doing something like existing one based on component you are using?
![image](https://github.com/bcgov/nr-fom/assets/81595625/2f6eee9b-fef4-42cf-a9e7-a6935fb7866a)

Reply:
yeah I'll add it, just want to make sure, thanks!